app/parse.py

```diff
@@ -8,8 +15,45 @@ class Quote:
     tags: list[str]
 
 
+def parse_single_quote(quote_soup: BeautifulSoup) -> Quote:
+    return Quote(
```

Comment:
It might be better to move all parsing functions to a separate file and use parse.py only to call that functions(just a suggestion)